website/content/installation/_index.md

```diff
@@ -145,6 +159,36 @@ configInline:
      - 198.51.100.0/24
 ```
 
+{{% notice note %}}
+If you want to deploy MetalLB using the [experimental FRR mode](https://metallb.universe.tf/configuration/#enabling-bfd-support-for-bgp-sessions), the following value must be set:
+
+```yaml
+speaker:
+  frr:
+    enabled: true
+```
+
+{{% /notice %}}
+
+## Using the MetalLB Operator
+
+The MetalLB Operator is available on OperatorHub. It eases the deployment and life-cycle of MetalLB in a cluster and allows configuring MetalLB via CRDs.
```

Comment:
I see you did not add a link for OperatorHub to https://operatorhub.io/operator/metallb-operator. Is there any reason? As a user, I would appreciate a direct link instead of searching for the OperatorHub website and then finding the MetalLB Operator page.

Reply:
Done